assets/tiller-app/resources/resources.yaml

```diff
@@ -51,3 +52,11 @@ spec:
               port: 44135
             initialDelaySeconds: 1
             timeoutSeconds: 1
+---
+apiVersion: scheduling.k8s.io/v1beta1
+kind: PriorityClass
+metadata:
+  name: tiller-high-priority
+value: 1000000
```

Comment:
I'm not sure I see tiller as something like a critical cluster task, so I'd consider dropping it down a couple points. 

Also, I wonder if we should consider setting a globalDefault.